src/app/workflow/workflow.component.html

```diff
@@ -276,19 +276,18 @@ <h3>
               </app-versions-workflow>
             </div>
             <ng-template #noVersionsVersionTab>
-              <div class="p-3">
+              <div [ngSwitch]="workflow.mode" class="p-3">
                 <mat-card class="alert alert-info" role="alert">
                   <mat-icon>info</mat-icon>
-                  <span *ngIf="entryType === EntryType.Service; else noVersionsVersionTabBioWorkflow">
-                    This service does not have any versions. Create a release/tag on GitHub with a valid .dockstore.yml to add a new
-                    version.
+                  <span *ngSwitchCase="WorkflowModel.ModeEnum.HOSTED"> To see versions, please add a new version in the Files tab. </span>
```

Comment:
Shouldn't these `*ngSwitchCase` be immediate children of the `ngSwitch`? It doesn't seem right to have lines 280-281 in between. Maybe I'm comparing this too directly to Java.